config.c

```diff
@@ -1443,6 +1445,26 @@ static int readConfigFile(const char *configFile, struct logInfo *defConfig)
                         }
                     } else if (!strcmp(key, "hourly")) {
                         set_criterium(&newlog->criterium, ROT_HOURLY, &criterium_set);
+                    } else if (!strcmp(key, "minutes")) {
+                        char *opt = key;
+
+                        key = isolateValue(configFile, lineNum, opt, &start, &buf, length);
+                        if (key && key[0]) {
+			    unsigned int minutes = (unsigned int)strtoul(key, &chptr, 0);
+                            if (*chptr || minutes <= 0) {
```

Comment:
I know this line is copied from surrounding code, but I prefer an explicit check for readability:

```c
*chptr != '\0'
```

Maybe we should also check for an upper minute limit, like `1440` (1 day)?

Reply:
update the check for *chptr. About the upper limit, I was hoping for minutes to be a very generic and flexible config option allowing configuration of duration since the step between hourly and daily and then daily and weekly and then weekly and month is very huge. Hence, I am hoping that not having a upper limit of 1 day to minutes is acceptable.